AllReadyApp/Web-App/AllReady/Controllers/EventController.cs

```diff
@@ -46,7 +51,7 @@ public IActionResult Index()
         [AllowAnonymous]
         public async Task<IActionResult> ShowEvent(int id)
         {
-            var viewModel = await _mediator.SendAsync(new ShowEventQueryAsync { EventId = id, User = User });
+            var viewModel = await _mediator.SendAsync(new ShowEventQuery { EventId = id, UserId = await User.GetUserId(_userManager) });

```

Comment:
I'm not sure you need your GetUserId extension. The userManager has a GetUserId taking a ClaimsPrinciple which I think you could use e.g. _userManager.GetUserId(User);